### PR TITLE
US20478 Group Detail Page

### PR DESCRIPTION
**Pull request by silasbaur**
## Problem
Create new group detail page based off of [new design](https://invis.io/6BZOS8ZSE2Y#/440779330_03-03b-Site-Based-Groups-Internal)

## Solution
[Preview](https://deploy-preview-1987--int-crds-net.netlify.app/groups/onsite/fathers/mason/)

## Corresponding PRs
**👇 This needs to be merged first**
[**Crds-Styles**](https://github.com/crdschurch/crds-styles/pull/502)

### CODE REVIEW

File: _layouts/onsite-groups/detail.html [hunk 1]
```diff
@@ -2,47 +2,61 @@
...
-  <div class="row push-bottom push-top">
-    <div class="col-md-12">
-      <h3 class="font-family-condensed-extra flush-ends text-uppercase">
-        {{ page.group.title }}
-      </h3>
-    </div>
-  </div>
+  <section class="container bg-white text-gray-dark soft-ends">
+    {% if page.group.length %}
+    <div class="push-bottom font-size-small"><strong>How Long</strong><br> {{ page.group.length }}</div>
```
Comment (kellgauthier) [inline _layouts/onsite-groups/detail.html:29]: Should be How Long:

File: _includes/groups/_meeting-details-card.html [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+      Online via Zoom
+    </p>
+    {% endif %}
+  
+    {% if meeting.childcare %}
+    <p class="font-size-smaller text-gray">* childcare is available with one week advance registration</p>
+    {% endif %}
+    
+    {% if meeting.registration_link %}
+    <a class="btn btn-orange btn-outline" href="{{ meeting.registration_link }}" type="button">Register</a>
```
Comment (kellgauthier) [inline _includes/groups/_meeting-details-card.html:32]: Per design, should be dark orange

**Review by kellgauthier — CHANGES_REQUESTED** [review]
I left a few inline comments, also margin should be adjusted. 

Preview
Screen Shot 2021-01-06 at 10 15 06 AM

Design
Screen Shot 2021-01-06 at 10 15 12 AM

File: _includes/groups/_meeting-details-card.html (hunk 2 above)
Comment (silasbaur) [reply _includes/groups/_meeting-details-card.html:32]: Dark Orange is not an option yet. We should keep it the original orange unless we keep seeing a pattern of the dark orange button in the future.

**Review by kellgauthier — APPROVED** [review]

**Pull request merged**